core/commands/blame.py

```diff
@@ -71,8 +72,7 @@ def picked_commit(self, commit_hash):
         super().run()
 
 
-
-class GsBlameInitializeViewCommand(TextCommand, GitCommand):
+class GsBlameInitializeViewCommand(GsHandleVintageousCommand, TextCommand, GitCommand):
```

Comment:
I am sorry, thought of vintage support but apparently forgot. Sorry.
I think the issue is that either you need to call
`view.run_command("gs_handle_vintageous")` on line 65
Then you won't need to inherit it here.

This is who it's done in inline diff [core/commands/inline_diff.py#L84-L85](https://github.com/divmain/GitSavvy/blob/49ce7bdb83abc88485ef8a1582ad77a8e83e9ee8/core/commands/inline_diff.py#L84-L85)